docker-compose.yml

```diff
@@ -4,49 +4,109 @@ services:
         image: zquestz/bitcoin-unlimited
         container_name: bitcoind-regtest
         ports:
-          - "18332:18332"
+          - "18443:18443"
```

Comment:
I just test setup and run and it seems ./bitcoin-cli can't connect to server
<img width="1005" alt="Screen Shot 2021-01-04 at 12 58 27" src="https://user-images.githubusercontent.com/6429197/103505553-197e6080-4e8d-11eb-9949-7210ab6d37e3.png">

Reply:
Strange, although I don't think it's related to the port being changed. Technically 18332 is meant for testnet, and 18443 is meant for regtest. I'll test things out a bit more and see if I can reproduce this issue.